docs/guide/content_rendering.md

```diff
@@ -1013,27 +1013,25 @@ With this configuration, the following controller will forward the request to th
 
 namespace Acme\TestBundle\Controller;
 
-use Symfony\Component\HttpFoundation\Response;
 use eZ\Bundle\EzPublishCoreBundle\Controller;
+use eZ\Publish\Core\MVC\Symfony\View\ContentView;
 
 class DefaultController extends Controller
 {
-    public function articleViewEnhancedAction( $locationId, $viewType, $layout = false, array $params = array() )
+    public function articleViewEnhancedAction(ContentView $view)
     {
         // Add custom parameters to existing ones.
-        $params += array( 'myCustomVariable' => "Hey, I'm a custom message!" );
-        // Forward the request to the original ViewController
-        // And get the response. Possibly alter it (here we change the smax-age for cache).
-        $response = $this->get( 'ez_content' )->viewLocation( $locationId, $viewType, $layout, $params );
-        $response->setSharedMaxAge( 600 );
+        $view->addParameters(['myCustomVariable' => "Hey, I'm a custom message!"]);
 
-        return $response;
+        // If you wish, you can also easily access Location and Content objects
+        $location = $view->getLocation();
+        $content = $view->getContent();
```

Comment:
comment out these two lines

Then add section below with something like so it is clear it is examples:
```
// Disable Content View cache
// Preferred alternative if possible: Vary by cookie, with default VCL it implies session cookie only
// $view->setCacheEnabled(false);

// Set as embed (normally this is done for you)
// $view->setIsEmbed(true);

// Set custom Response variables (for instance custom headers)
//$view->setResponse((new Response())->setExpires(new DateTime('@946684800')));
```

Please test last option tough, unsure how it works, of content and status code will be set correctly, and if some of our listeners/VCL might override things like expiry.

Reply:
I have commented out these two lines. I haven't added the section below yet because we need to make sure that it makes sense as an example here.